### ISSUE TITLE
Acceptable Conduct: Attendance 

### ISSUE THREAD

**Issue by MAbraham24** [burgess01/code-of-conduct#2]
I worked on the Attendance section of Acceptable Conduct. I created four different bullet points regarding acceptable conduct for attendance, but I ran into issues with wording. For my first bullet point I said "Students should arrive to class on time". The issue with this is that the wording seems to vague and could be better expanded upon, so that there is no possible loopholes. How can I fix this?

**Comment by tuduun** [comment]
Obviously, we need to respect others' time and make it in time for class. But, maybe try adding another bullet point after that saying that let the professor know if you would be late to class. That way it does not sound that harsh.

**Comment by burgess01** [comment]
Maybe 'Students should arrive to class on time and be ready to start class at 1040am"? it is a bit more specific and would prevent people from walking in the door and not being ready when the professor begins her presentations.

**Comment by laurennevill** [comment]
To the last point, I would adjust "student who are late" to "students who are late or absent" so it covers issues of absence and tardiness alike.

**Comment by MAbraham24** [comment]
> Obviously, we need to respect others' time and make it in time for class. But, maybe try adding another bullet point after that saying that let the professor know if you would be late to class. That way it does not sound that harsh.

Thanks for the input! I will work towards implementing your advice.

**Comment by MAbraham24** [comment]
> 

I feel like this a great change, I will make it right now!

**Comment by MAbraham24** [comment]
> To the last point, I would adjust "student who are late" to "students who are late or absent" so it covers issues of absence and tardiness alike.

I will combine this with what Katie recommended, thank you so much for the input!
